NSDate+Extension.swift

```diff
@@ -9,65 +9,65 @@
 
 import Foundation
 
-func NSDateTimeAgoLocalizedStrings(key: String) -> String {
+func NSDateTimeAgoLocalizedStrings(_ key: String) -> String {
     let resourcePath: String?
 
-    if let frameworkBundle = NSBundle(identifier: "com.kevinlawler.NSDateTimeAgo") {
+    if let frameworkBundle = Bundle(identifier: "com.kevinlawler.NSDateTimeAgo") {
         // Load from Framework
         resourcePath = frameworkBundle.resourcePath
     } else {
         // Load from Main Bundle
-        resourcePath = NSBundle.mainBundle().resourcePath
+        resourcePath = Bundle.main.resourcePath
     }
 
     if resourcePath == nil {
         return ""
     }
 
-    let path = NSURL(fileURLWithPath: resourcePath!).URLByAppendingPathComponent("NSDateTimeAgo.bundle")
-    guard let bundle = NSBundle(URL: path) else {
+    let path = URL(fileURLWithPath: resourcePath!).appendingPathComponent("NSDateTimeAgo.bundle")
+    guard let bundle = Bundle(url: path) else {
         return ""
```

Comment:
Returning the ```key``` instead of an empty string, along with printing some warning, would be a more sensible default. It might work even if the bundle is missing.